chapters/hardware-software-interface/structures/reading/README.md

```diff
@@ -0,0 +1,97 @@
+# Structured Data (Structures, Arrays and String Operations)
+
+In this laboratory, we will introduce the concept of structures in assembly language and work with specialized operations on strings.
```

Comment:
```suggestion
We will introduce the concept of structures in assembly language and work with specialized operations on strings.
```
Don't mention "laboratory" because other teachers might use different terms or organise things differently.